hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -49,6 +49,13 @@ objects:
         namespace: openshift-custom-domains-operator
       spec:
         image: ${REGISTRY_IMG}@${IMAGE_DIGEST}
+        grpcPodConfig:
```

Comment:
This is only available in 4.11+ https://docs.openshift.com/container-platform/4.11/rest_api/operatorhub_apis/catalogsource-operators-coreos-com-v1alpha1.html